topbar.js

```diff
@@ -532,7 +530,7 @@ function enable () {
             return;
         fixTopBar();
     });
-
+    
```

Comment:
```suggestion

```

Whitespace error.

Reply:
Implemented in [4d0eab4](https://github.com/paperwm/PaperWM/pull/476/commits/4d0eab4d073588bcaba8a434e3eb88c9866541af).